src/components/shell/timeIntervalDropdown/timeIntervalDropdown.scss

```diff
@@ -5,35 +5,21 @@
 
 .time-interval-dropdown {
   @include rem-fallback(width, 200px);
+  .time-interval-dropdown-select {
+    height: 100%;
 
-  .Select-menu-outer {
-    margin-top: 1px !important;
-    border-top: none !important;
-  }
-
-  .Select-control {
-    border: none !important;
-    height: 100% !important;
+    @include themify($themes) {
+      // match the background color of the underlying panel:
+      background-color: themed('color-bg-content');
 
-    .Select-placeholder {
-      display: flex;
-      align-items: center;
-    }
-
-    .Select-input {
-      height: 100%;
-
-      input { height: 100%; }
+      &:not(:active), &:not(:hover), &:not(:focus) {
+        border-top-color: transparent;
```

Comment:
these should probably be added to the theme file

Reply:
Agreed, let's add it in when Ted give us the comps for dropdowns in an action bar.